scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3851,431 @@ def rvs(self, dim, size=1, random_state=None):
 
 
 unitary_group = unitary_group_gen()
+
+_mhg_doc_default_callparams = """\
```

Comment:
It looks like this is not your fault, but I think it is confusing that `x` shows up here.
![image](https://user-images.githubusercontent.com/6570539/93002148-707cb600-f4e9-11ea-9aca-63836d14e0ed.png)

It does for multinomial, too, but I don't think it's right.  You can't pass `x` into `multinomial`. And if `x` here is supposed to document `x` that gets passed into methods like `pmf`, then why don't we document the `size` parameter that gets passed into `rvs` here, too? Also, it seems like there is a keywork `seed` that is documented as `random_state`, right? Again, `multinomial` has the same issue, but we shouldn't perpetuate it here.

Reply:
> It looks like this is not your fault, but I think it is confusing that `x` shows up here

Yup, it shouldn't be like that. Changing it

Reply:
> Not sure what to do about all this yet. But do you agree that these things are confusing?

definitely! I think those are old docs that need to be changed...

I looked at other distribution's doc suite. Almost all of them have the quantiles `x` documentation in the `__init__` method itself. Is that an inconsistency too or does scipy do that on purpose?

Reply:
I assume one person did it by mistake (or for some historical reason) and everyone else has blindly copied since then. It sure doesn't seem like many people pay attention to the rendered documentation when we find all these other things that are certainly mistakes. (e.g. `random_state` vs `seed`, the fact that method documentation is inaccessible and the names render with backticks).
![image](https://user-images.githubusercontent.com/6570539/93002878-e8011400-f4ee-11ea-94b7-c3de8f5de1a1.png)

Reply:
Yes. I think most people, like me, are afraid to change the well-established code and mistakes propagate perpetually until spotted. Should these inconsistencies be addressed here or in some other pr?

Reply:
another PR.